crypto/threads_win.c

```diff
@@ -66,8 +61,9 @@ int CRYPTO_THREAD_read_lock(CRYPTO_RWLOCK *lock)
 {
 # ifdef USE_RWLOCK
     CRYPTO_win_rwlock *rwlock = lock;
+
     AcquireSRWLockShared(&rwlock->lock);
-    rwlock->mode = SRWLOCK_MODE_SHARED;
```

Comment:
Just use 1 and 0 for TRUE and FALSE.

I can only see one instance of TRUE in the codebase and it is related to win32 code that uses BOOL ret;